### PR TITLE
Fix authlib.oauth2.rfc9068.JWTIntrospectionEndpoint documentation mistakes

### PR DESCRIPTION
**Pull request by twalcari**
A small fix to the documentation of `authlib.oauth2.rfc9068.JWTIntrospectionEndpoint`

**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

---

- [x] You consent that the copyright of your pull request source code belongs to Authlib's author.

### CODE REVIEW

**Comment by twalcari** [comment]
Hi @azmeuk , can you clarify why the documentation of `authlib.oauth2.rfc9068.JWTIntrospectionEndpoint` mentioned:
```python
authorization_server.register_endpoint(MyRefreshTokenIntrospectionEndpoint)
```

That class `MyRefreshTokenIntrospectionEndpoint` is not mentioned in the code example. Does that line need to be removed, or must that class `MyRefreshTokenIntrospectionEndpoint` be added to the documentation?

The same issue arises for the `authlib.oauth2.rfc9068.JWTRevocationEndpoint` by the way.

File: authlib/oauth2/rfc9068/introspection.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ class JWTIntrospectionEndpoint(IntrospectionEndpoint):
 
     ::
 
-        class MyJWTAccessTokenIntrospectionEndpoint(JWTRevocationEndpoint):
```
Comment (azmeuk) [inline authlib/oauth2/rfc9068/introspection.py:23]: This one is definitively an error. :+1:

**Comment by azmeuk** [comment]
The `MyRefreshTokenIntrospectionEndpoint` is not mentionned in the code, I simply added this to illustrate that there can be several introspection endpoints for different kinds of token (jwt or refresh for instance).

If several introspection endpoints are registered with `register_endpoint`, the first one will be executed. If a `ContinueIteration` exception is raised, then the second registration endpoint is executed and so on. Practically, when `JWTIntrospectionEndpoint` gets a refresh token, it passes it to the next registered introspection endpoint by raising a `ContinueIteration`.

https://github.com/azmeuk/authlib/blob/d589d4ff513a90168118f7bdec00b2fcaac49f41/authlib/oauth2/rfc9068/introspection.py#L63-L65

This behavior has been introduced with #576 but not documented. I am opened to ideas for making the documentation clearer about this.

**Comment by azmeuk** [comment]
I merged a part of your fix and slightly made the example clearer.

**Pull request closed without merging**